kapacitor/kapa_client.go

```diff
@@ -0,0 +1,103 @@
+package kapacitor
+
+import (
+	"sync"
+
+	client "github.com/influxdata/kapacitor/client/v1"
+)
+
+const ListTaskWorkers = 4
```

Comment:
We could, but it's tricky to dynamically figure out what would perform well. This workload is mostly I/O bound so it's dependent more on things like open file limits, bandwidth limits, etc. Some of that is visible to us, some isn't.

I started with 10 when running benchmarks and found that 4 performed better after watching some execution traces. There is less rescheduling of different goroutines, and there's still space for additional request goroutines and the GC thread to run.

I basically decided to start with a "good enough" figure and if the performance is bothering someone we can revisit on a new issue.